### PR TITLE
Add some notes about when to use (or not) Doctrine listeners

### PR DESCRIPTION
**Pull request by javiereguiluz**
This morning we discussed about this in the Symfony Slack and I thought we could add some notes in the official Doctrine docs too. Thanks for your consideration.

### CODE REVIEW

File: docs/en/reference/events.rst [hunk 1]
```diff
@@ -6,6 +6,10 @@ Common package. Doctrine uses it to dispatch system events, mainly
 :ref:`lifecycle events <reference-events-lifecycle-events>`.
 You can also use it for your own custom events.
 
+Doctrine event listeners are intended for persistence hooks (i.e. *"save also
+this when saving that"*). They should not be used for domain logic, such as
+logging changes, setting updatedAt and createdAt properties, etc.
```
Comment (greg0ire) [inline docs/en/reference/events.rst]:
```suggestion
logging changes, setting ``updatedAt`` and ``createdAt`` properties, etc.
```

**Review by greg0ire — APPROVED** [review]

**Comment by greg0ire** [comment]
Triggered a new build

**Review by SenseException — APPROVED** [review]

**Comment by lcobucci** [comment]
Shouldn't this use `2.6` as base instead of `master`? It's quite relevant and applicable to our current stable version IMHO.

**Comment by javiereguiluz** [comment]
I'm closing this old PR because I won't have time to rebase it. Sorry!

**Pull request closed without merging**